hi-michelleprimiani/pixo-client
# Created utils folder - FormInput.jsx

## Add FormInput.jsx Component for Reusable Form Fields 

This PR introduces the 'FormInput' component, designed for reusability for form input patterns across the site. The goal is to reduce redundancy and increase maintainability by provided an interface for various form inputs such as text, URL, textarea and checkbox. This aligns with my effort to follow DRY principles and streamline our form implementations. 

## Changes 

- FormInput Component: Created a new ```FormInput.jsx``` that handles text, textarea, url and checkbox inputs
- Styling: Incorporated Tailwind CSS for styling
- Integration: Replaced raw input elements in the ```CreateCollectibleForm``` and ```EditCollectibleForm``` with the new FormInput component and start the refactoring process.

## How To Test 
- Navigate to any form that has been updated to use FormInput.
- Fill out the form, ensuring that inputs behave as expected.
- Submit the form and verify that the data is processed correctly.
